Bug in ci/steps.yml

There's a bug in the `Run Aqt` step that causes jobs to fail when they really shouldn't.

Here's the relevant code:

```bash
set -ex
number=$RANDOM
let "number %= 60"
```

Here's a copy of [the log from a recent run](https://dev.azure.com/miurahr/github/_build/results?buildId=4195&view=logs&j=3a25022a-f08c-5c04-be7d-d279a7fc48ef&t=7e2ee2cf-8346-573d-e7b5-4588bb42191e):
```
+ number=4980
+ let 'number %= 60'
##[error]Bash exited with code '1'.
Finishing: Run Aqt
```

Note that 4980 is a multiple 60, so `4980 % 60` is 0.

In Bash, `let 'number = 1 % 60'` returns 0, showing that the command succeeded. However, `let 'number = 0 % 60'` returns 1, which signals to bash that the command failed. Since `set -e` was used earlier in the script, the script exits early with code '1'.

~~I'm not sure what the best solution to this is. I think there are ways to cancel out the '1' return code using the 'or' operator, but it's not working for me right now. Since this error comes right at the beginning of the script, you could probably move `set -e` to somewhere after the modulus operation. I think that might be considered bad practice though.~~
One solution is: 

```bash
let 'number %= 60' || true
```

--- a/ci/steps.yml
+++ b/ci/steps.yml
@@ -14,7 +14,7 @@ steps:
   - bash: |
       set -ex
       number=$RANDOM
-      let "number %= 60"
+      let "number %= 60" || true
       let "number += 1"
       sleep $number
       mkdir Qt

